uds/__init__.py

```
@@ -20,14 +20,15 @@
     "can",
-    "client",
-    "translator",
     "diagnostic_configuration",
     "message",
     "packet",
     "segmentation",
+    "client",
+    "translator",
     "transport_interface",
     "utilities",
+    "client",
```

Ruff (0.16.1)

[warning] `__all__` contains duplicate entries: `client` duplicated here

Remove duplicate entries from `__all__`

(RUF068)